saleor/plugins/manager.py

```diff
@@ -1292,6 +1283,36 @@ def __run_payment_method(
             " payment method is inaccessible!"
         )
 
+    def __run_tax_method(
```

Comment:
It looks like a generic method for calling sync action that returns first sucess response. I don't think that it need to be dedicated to taxes. 
The `if isinstance(result, TaxData)` is not really needed right? We can assume that we return result when `if result is not None`, right?